### ISSUE TITLE
False offense Layout/IndentHash

### ISSUE THREAD

**Issue by akkee** [rubocop-hq/rubocop#6528]
`rubocop 0.60.0` `ruby 2.5.3` `rails 5.2.1`

this is the code snippet:

```rb
  def markdown(text)
  render_options = {
    # will remove from the output HTML tags inputted by user
    filter_html: true,
    # will insert <br /> tags in paragraphs where are newlines
    # (ignored by default)
    hard_wrap: true,
    # hash for extra link options, for example 'nofollow'
    link_attributes: { rel: 'nofollow' }
    # more
    # will remove <img> tags from output
    # no_images: true
    # will remove <a> tags from output
    # no_links: true
    # will remove <style> tags from output
    # no_styles: true
    # generate links for only safe protocols
    # safe_links_only: true
    # and more ... (prettify, with_toc_data, xhtml)
  }

  coderayified = CodeRayify.new(render_options)

  extensions = {
    # will parse links without need of enclosing them
    # autolink:           true,
    # blocks delimited with 3 ` or ~ will be considered as code block.
    # No need to indent.  You can provide language name too.
    # ```ruby
    # block of code
    # ```
    fenced_code_blocks: true,
    # will ignore standard require for empty lines surrounding HTML blocks
    lax_spacing:        true,
    # will not generate emphasis inside of words, for example no_emph_no
    no_intra_emphasis:  true,
    # will parse strikethrough from ~~, for example: ~~bad~~
    strikethrough:      true,
    # will parse superscript after ^, you can wrap superscript in ()
    superscript:        true,
    # will require a space after # in defining headers
    space_after_headers: true,
    # Parse tables
    tables: true,
    # disable indented code blocks
    disable_indented_code_blocks: true,
    no_links: true,
    prettify: true
  }

  _markdown_to_html = Redcarpet::Markdown.new(coderayified, extensions).render(text).html_safe
end
```

## Expected behavior

`Layout/IndentHash` should not throw offences as the hash indentation is exactly as needed by the default setting `special_inside_parentheses`.

## Actual behavior

I get these offences listed:

```sh
  lib/ext/redcarpet_coderay.rb:12:5: C: Layout/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.
    filter_html: true,
    ^^^^^^^^^^^^^^^^^
lib/ext/redcarpet_coderay.rb:40:5: C: Layout/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.
    fenced_code_blocks: true,
```

## Steps to reproduce the problem

Copy the above code in an rb file and run rubocop for `Layout/IndentHash` offence

## RuboCop version

```
$ [bundle exec] rubocop -V
0.60.0 (using Parser 2.5.3.0, running on ruby 2.5.3 x86_64-darwin17)
```
